src/selectors/tracks/index.js

```diff
@@ -4,15 +4,18 @@ import uniq from 'lodash/uniq';
 
 import { TRACK_LENGTH_ORIGINS } from '../../ducks/tracks';
 
-import { getTimeOfDayPeriodBasedOnTime, trimTrackDataToTimeRange } from '../../utils/tracks';
+import {
+  trimTrackDataToTimeRange,
+  buildFeatureCollectionOfTwoPointLineStringSegments
```

Comment:
I think Luis already mentioned, but `buildFeatureCollectionOfTwoPointLineStringSegments` is a pretty stanky and confusing name